gbpservice/nfp/pecan_api/api/root_controller.py

```diff
@@ -12,16 +12,25 @@
 
 import pecan
 
-from v1 import controllers
+import pecan
 
+try:
+    from gbpservice.tests.contrib.nfp_service.reference_configurator\
```

Comment:
One of the comments from GBP folks is to avoid this "\" and replace with "(". Make this change in all the places.